### PR TITLE
Adding new games FGO and GFL

### PR DESCRIPTION
**Pull request by namv22**
FGO US v2.22.1 (135) and Girls' Frontline EN v2.0702_362 (362)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,6 +122,7 @@ wsa://com.android.settings
 | Formula 1 | 11.0.1449 | ✅ | |
 | FAST Speed Test | 1.0.8 (88) | ✅ |||
 | Fancade | 1.7.6 | ❌ | App crashes ||
+| Fate/Grand Order (US) FGO | 2.22.1 (135) | ✅ || AMD system: A little unstable, but playable |
```
Comment (pratyakshm) [inline README.md:125]:
```suggestion
| Fate/Grand Order (US) FGO | 2.22.1 (135) | ✅ || A little unstable, but playable |
```

File: README.md [hunk 2]
```diff
@@ -136,6 +137,7 @@ wsa://com.android.settings
 | Genshin Impact (Cloud app) || ✅ |||
 | Geekbench |5.4.1| ✅ |||
 | GeoGebra | 5.0.674.0 | ✅ | | |
+| Girls' Frontline (EN) | 2.0702_362 (362) | ⚠️ || Unconfirmed: Game will always perma freezes after the plash screen, already tried restarting the PC but doesn't work. After installing WSL2, game can now run. AMD system PC |
```
Comment (pratyakshm) [inline README.md:140]:
```suggestion
| Girls' Frontline (EN) | 2.0702_362 (362) | ⚠️ || Game freezes after splash screen, works after installing WSL2  |
```

Can you please clarify what exactly do you mean when you say installing WSL2? Is it the same as any other WSL deployment using ``wsl --install``, or have you tried something extra?

**Review by pratyakshm — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (namv22) [reply README.md:140]: Yes, sorry for that, I only use `wsl --install` and that is, after I installed it yesterday, the game actually runs today, so I'm a little confused

**Review by namv22 — COMMENTED** [review]
I changed the description as per suggestion

**Comment by pratyakshm** [comment]
Thanks!

**Pull request merged**